modules/org/infinispan/dependencies/jdk/openjdk/trim.sh

```diff
@@ -1,5 +1,16 @@
 #!/bin/bash
-set -e
+#set -e
 DIR=$(dirname $0)
 PACKAGES=$(cat $DIR/packages.txt)
-rpm -e --nodeps $PACKAGES
+for pkg in $PACKAGES
+do
+    cmd=`rpm --verify $pkg 2> /dev/null`
+    rc=$?
+    if [[ $rc -eq 0 ]]
+    then
+        echo "${pkg} exists"
+        rpm -e --nodeps $pkg
+    else
+        echo "${pkg} does not exist"
```

Comment:
The `rpm --verify` command already prints the following out to stdout "package %s is not installed", and stdout is not captured by `2> dev/null`. Therefore, we can just remove this else statement and the output will be the following if we have two packages, one which exists and one which does not:

```
%s exists
package %s is not installed
```